zulip/integrations/matrix/test_matrix.py

```diff
@@ -73,3 +75,19 @@ def test_write_sample_config(self):
 
             with open(path) as sample_file:
                 self.assertEqual(sample_file.read(), sample_config_text)
+
+
+class MatrixBridgeZulipToMatrixTests(TestCase):
+    def test_zulip_message_validity_success(self):
+        # type: () -> None
+        zulip_config = dict(stream="some stream",
+                            topic="some topic",
+                            email="some@email")
+        msg = dict(
+            sender_email="John@Smith.smith",  # must not be equal to config:email
+            type="stream",  # Can only mirror Zulip streams
+            display_recipient=zulip_config['stream'],
+            subject=zulip_config['topic']
+        )
+        assert msg['sender_email'] != zulip_config['email']
```

Comment:
I don't think this is needed, as the `msg` and `zulip_config` are just defined above.

Reply:
This is to ensure that they are different.